genai-perf/tests/test_llm_profile_data_parser.py

```diff
@@ -41,6 +41,97 @@ def ns_to_sec(ns: int) -> Union[int, float]:
     return ns / 1e9
 
 
+def create_stats_dict(
```

Comment:
Talked offline--Hyunjae will attempt to use the pre-existing production code Statistics class to hold this information for the actual/observed data for the test.

Since no comparator functionality exists in the production code Statistics class, but one is needed for testing, a comparator function could be added just to this test file to handle that.